extensions/panache/mongodb-panache/deployment/src/main/java/io/quarkus/mongodb/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -35,7 +37,12 @@
 
     private static final DotName DOTNAME_OBJECT_ID = DotName.createSimple(ObjectId.class.getName());
 
-    @BuildStep(providesCapabilities = "io.quarkus.mongodb.panache")
+    @BuildStep
+    CapabilityBuildItem capability() {
+        return new CapabilityBuildItem(Capabilities.MONGO_PANACHE);
```

Comment:
It's very minor but could we call it `MONGODB_PANACHE` as the feature? See a few lines below.

Reply:
I had some spare cycles waiting for CI so I fixed it and fixed another typo on the OAUTH2 constant.

I just force pushed.